strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/maven/MavenArtifactControllerTest.java

```diff
@@ -682,27 +689,12 @@ public void whenRepositoryIsOutOfServiceWeShouldDisallowArtifactDeployment()
     }
 
     @Test
-    public void testNonExistingDirectoryDownload()
+    public void testDirectoryDownload()
```

Comment:
we probably also need a test case with invalid file path, like it was `/storages/storage-common-proxies/maven-central/john/doe/who.jar` .

Reply:
I restored `MavenArtifactControllerTest#testNonExistingArtifactInNonExistingDirectory` with `/storages/storage-common-proxies/maven-central/john/doe/who.jar` path but the error is 400 now, because of new interceptor validation.